src/backend/tasks_io/datafeeds/datafeed_fms_api.py

```diff
@@ -72,6 +72,8 @@ class DatafeedFMSAPI:
         "dar": "darwin",
         "gal": "galileo",
         "hop": "hopper",
+        "john": "johnson",
```

Comment:
Statbotics is using `mil` and `joh`. should we do the same?

Reply:
I'm good either way; I can get this updated.